### PR TITLE
Stm32 huk

### PR DESCRIPTION
**Pull request by etienne-lms**
_stm32mp15_huk.c_ updates following discussion in https://github.com/OP-TEE/optee_os/pull/5555.

### CODE REVIEW

File: core/drivers/stm32mp15_huk.c [hunk 1]
```diff
@@ -15,6 +15,21 @@
 #include <string.h>
 #include <string_ext.h>
 
+#if defined(CFG_STM32MP15_HUK_BSEC_KEY) && \
```
Comment (jenswi-linaro) [inline core/drivers/stm32mp15_huk.c:18]: This kind of stuff is usually done in some .mk file instead.

File: MAINTAINERS [hunk 2]
```diff
@@ -289,5 +292,4 @@ M:	Jerome Forissier <jerome.forissier@linaro.org> [@jforissier]
 R:	Etienne Carriere <etienne.carriere@linaro.org> [@etienne-lms]
 L:	op-tee@lists.trustedfirmware.org
 S:	Maintained
-F:	*
-F:	*/
+FNF:	*/
```
Comment (jenswi-linaro) [inline MAINTAINERS:295]: This isn't mentioned in the commit message.

**Review by jenswi-linaro — COMMENTED** [review]
`Acked-by: Jens Wiklander <jens.wiklander@linaro.org>`

**Review by zengxx-HUAWEI — APPROVED** [review]

**Comment by etienne-lms** [comment]
@xiaoxuZeng, do you wan tI add your `Acked-by` tag?

**Comment by etienne-lms** [comment]
Fixup commits squashed and tag applied.

**Pull request merged**